Source/WebKit/GPUProcess/mac/com.apple.WebKit.GPUProcess.sb.in

```diff
@@ -854,6 +854,9 @@
     (global-name "com.apple.coremedia.routediscoverer.xpc")
 #if __MAC_OS_X_VERSION_MIN_REQUIRED > 120000
     (global-name "com.apple.coremedia.samplebufferconsumer.xpc")
+#endif
+#if __MAC_OS_X_VERSION_MIN_REQUIRED >= 130000
+    (global-name "com.apple.coremedia.shaphelper.xpc")
```

Comment:
Since this only seems to be needed for AirPlay, I wonder if we should try to detect that in the UI process, and issue an extension. Does the UI process know when AirPlay is being started?

Reply:
It seems like it may be used for SharePlay, too; I'm not sure how that works with WebKit, and whether any AirPlay bits are active for that use case.

Reply:
I see, thanks! Perhaps we can file a follow-up bug to address services in GPUP which are only needed for AirPlay/SharePlay? If this service meets our criteria for being added to the GPUP sandbox, I think this looks good.